src/sys/unix.rs

```diff
@@ -1597,6 +1597,93 @@ impl crate::Socket {
         ))
         .map(|_| sbytes as usize)
     }
+
+    /// Set the value of the `TCP_USER_TIMEOUT` option on this socket.
+    ///
+    /// If set, this specifies the maximum amount of time that transmitted data may remain
+    /// unacknowledged or buffered data may remain untransmitted before TCP will forcibly close the
+    /// corresponding connection.
+    #[cfg(all(
+        feature = "all",
+        any(
```

Comment:
You can reduce this to Android, Fuchsia and Linux. So we have don't have any special targets for emscripten or l4re. Same for the other `cfg` attributes (also in the test).

Reply:
https://github.com/rust-lang/socket2/pull/241/commits/7b8473b96a2da2b15a01eb6e91aedf04ce354afd